src/ng/directive/as.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name as
+ * @restrict A
+ *
+ * @description
+ * Publishes a component controller or a DOM element into the current scope.
+ *
+ * @element ANY
+ * @param {expression} as {@link guide/expression Expression} to assign the controller.
+ *
+ * This directive allows to access to the controller of components in your
+ * current template or from current controller.
+ *
+ * @example
+ * ### Access to component controller
+ * This example demonstrates basic publishing of the component controller into the scope.
+ * <example name="asDirectiveComponentExample" module="asComponentExample">
+ *   <file name="index.html">
+ *     <script>
```

Comment:
Ok, I'll correct it. It seems right.

El dv, 19 febr 2016 a les 13:59 Georgios Kalpakas <notifications@github.com>
va escriure:

> In src/ng/directive/as.js
> <https://github.com/angular/angular.js/pull/14080#discussion_r53457211>:
>
> > + *
> > + * @description
> > + * Publishes a component controller or a DOM element into the current scope.
> > + *
> > + * @element ANY
> > + * @param {expression} as {@link guide/expression Expression} to assign the controller.
> > + *
> > + * This directive allows to access to the controller of components in your
> > + * current template or from current controller.
> > + *
> > + * @example
> > + * ### Access to component controller
> > + * This example demonstrates basic publishing of the component controller into the scope.
> > + * <example name="asDirectiveComponentExample" module="asComponentExample">
> > + *   <file name="index.html">
> > + *     <script>
>
> Why not put this in a different script.js file ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/angular.js/pull/14080/files#r53457211>.
>